### ISSUE TITLE
Is this project not active any more? 

### ISSUE THREAD

**Issue by Montoya** [radiant/radiant#407]
I've been looking at RoR-based CMS's and Radiant seems to be the most popular one, but it looks like this project started moving to a 2.0 release a few years ago and then never made it. Should I be looking elsewhere if I want an actively maintained CMS solution?

**Comment by jlong** [comment]
The project has stalled a bit. We need more active contributors to keep it going.

Current Gem only runs on 1.9 and you must patch your Gemfile to get everything to install. We are working on a fix for this. The work in master will probably be moved into a branch. It was the Rails 4 upgrade path, but work stalled on that. @avonderluft is working on some quick fixes right now.

There are other CMS's that are more actively maintained at the moment. Me, I've got my fingers crossed 🤞  for a Radiant Resurrection.

If you are interested in lending a hand, I can add you to our Slack.

**Comment by Montoya** [comment]
Sorry, I'm not a Ruby developer, just a PM for a team that uses Ruby. But we are looking for something with minimal effort at the moment, so probably won't be able to contribute at all. Thanks for letting me know though! I do hope Radiant gets resurrected soon :)

**Comment by SamWhited** [comment]
> If you are interested in lending a hand, I can add you to our Slack.

As I said on the ML (but I know people don't read the ML that much, so I'll repeat it here): Please consider using something that doesn't require that I sign up for yet another account, or go through the process of getting an invite. I know Slack is the new hotness, but I see no benefit to using it over the existing solution (the IRC room); it's solving a problem that doesn't exist. I'm aware not everyone is as phobic about signing up for new accounts for things as I am, but I don't see why it's necessary in any way in this case.

**Comment by jlong** [comment]
Hey @SamWhited I don't see your comments on the mailing list, are you sure they went through To respond briefly, Slack is a convenience for the moment for members of the dev team and anyone else who wants to contribute to this effort. It this gets traction we might be able to consider an alternative. Right now I'm more concerned about traction than the tools we are using. Are you interested in contributing?

**Comment by SamWhited** [comment]
> Are you interested in contributing?

Yes, I'd definitely like help with any revamp work; I used to really like working on Radiant back in the day (although I deeply dislike Ruby and Rails, so we'll see, I'm not sure if I could make myself go back to that sort of thing).

**Comment by jlong** [comment]
Sounds great. Is Slack an obstacle to contributing for you? I think I sent you an invite earlier.

**Comment by SamWhited** [comment]
> Is Slack an obstacle to contributing for you?

Yes, I don't want to sign up for yet another account on yet another closed chat platform; however, if I'm really alone in this concern (and I suspect I am), I wouldn't want to ask people to change the tools they're comfortable with just for someone who may or may not end up actively contributing again.

**Comment by saturnflyer** [comment]
I personally feel that the mailing list is fine. I haven't been in IRC in ages but that would also be fine with me.
They are both public, whereas slack is not.

But contributing has always been possible so if anyone is truly not interested in Ruby or Rails then they likely won't contribute.

Sam, Your contributions were really fantastic in the past. I'm glad your interested in continuing.

**Comment by jlong** [comment]
I'm not much of a mailing lister these days. So much email. Happy for conversations to continue there though. I view Slack as more of a hangout. A place for informal conversations. Those conversations can happen on the mailing list, too, for sure. But email is more formal. Less relaxed.

**Comment by Montoya** [comment]
Not sure if this helps, but I think you can connect to Slack with IRC without having to go through the whole account-setup-process: http://www.pilgrimbreak.com/how-to-access-slack-with-command-line-irc-client-irssi/

**Comment by jlong** [comment]
@Montoya Nice info! Although it looks like that is a user-level preference, meaning that it would still require a Slack signup. I believe @SamWhited is trying to avoid signing up for Slack.

**Comment by jamesakers** [comment]
I am willing to contribute to a Radiant resurrection. It has served me well in the past.

**Comment by makjaveli** [comment]
Which branch is the most promising to get Radiant working with Rails 4.2 and Ruby >2.1? 

'Release' or 'Master'?

**Comment by jlong** [comment]
@avonderluft is working on the "Release" branch. The branch is focused on getting Radiant working again with a good Gemfile. Currently there are errors if you just try to install the normal way because some of the Gem version requirements were too loose. I believe he is having trouble getting some of the tests to pass on Travis. He could use some help.

@saturnflyer was working on getting Radiant going on a more modern version of Rails in the "master" branch, but I believe the effort has stalled. I know he would love help resurrecting this effort.

@jamesakers If you'd like to help I can give you commit access to the repo.

**Comment by jamesakers** [comment]
@jlong Yes, please add me.

**Comment by blj** [comment]
I am currently working on getting Radiant CMS back up and working again. If any one care to look at my [fork of Radiant CMS](https://github.com/blj/radiant), appreciate it. Also would like to have an invite to the Slack, so I can keep it going. I have most of the model specs passing now. Thanks.

**Comment by ronald** [comment]
I'm currently working on Radiant (Ruby 2.6, Rails 6 beta/edge).

I'd like to provide more pull request but I need some help on the ExtensionLoader (#421) and a local test setup.

Is anyone out there? 
Which channel (irc, maillinglist, slack, issues) should be used for chat?

**Comment by avonderluft** [comment]
Hi @ronald - our Slack has not been very active, but if you provide your email, I'll send you an invite.

**Comment by ronald** [comment]
@avonderluft, it's mr.ronald@gmx.de

**Comment by avonderluft** [comment]
slack invite sent.

**Comment by blj** [comment]
I would also like to help and get radiant back to stable. Let me know if
any specific area you want immediate help with.

@blj
